icons/lbry.svg

```diff
@@ -0,0 +1,4 @@
+<svg role='img' xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+<title>Lbry.tv icon</title><path d="M23.493 8.695v1.192l-11.734 7.21-8.842-4.336.016-.668 8.792 4.328 11.176-6.864v-.49l-10.55-5.09-11.768 7.28v3.254l11.142 5.512 11.633-7.135.33.507-11.929 7.304L0 14.883v-3.957l12.325-7.625z"/>
+  <path d="M23.35 14.19l.168-1.066-1.031-.177.067-.414 1.446.245-.237 1.48z"/>
+</svg>
```

Comment:
The build currently fails, because it expects the SVG to have only _one_ path element and everything on one line without line breaks.

Reply:
You can achieve this by using SVGO (or svgomg) – instead of doing it by hand. See the [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#3-optimize-the-icon).
Invoke with the comment `npm run svgo -- icons/lbry.svg`

Reply:
Couldn't find the way to do that using svgomg. The npm way worked though, thanks.